cpp/src/nvgraph_gdf.cu

```diff
@@ -515,6 +515,69 @@ gdf_error gdf_AnalyzeClustering_ratio_cut_nvgraph(gdf_graph* gdf_G,
 	return GDF_SUCCESS;
 }
 
+
+gdf_error gdf_extract_subgraph_vertex_nvgraph(gdf_graph* gdf_G,
+																							gdf_graph* result,
```

Comment:
Indentation is weird, here. The input parameter list should be updated to match the suggested changes in the header file.

Reply:
Converted tabs to spaces in the file.  Should address the weird indentation.